ctp2_code/gs/slic/SlicEngine.cpp

```diff
@@ -953,37 +940,29 @@ void SlicEngine::AddBuiltinFunctions()
 	m_functionHash->Add(new Slic_GetMostAtRiskCity);
 	m_functionHash->Add(new Slic_GetRoundsToNextDisaster);
 	m_functionHash->Add(new Slic_GetCurrentPollutionLevel);
-	// New slicfunction by The Big Mc
```

Comment:
These slic function below are not from CTP2.1, so you may prune the comments but you still need to make that clear that you do not find those in Activision's patch.